packages/contracts/test/SphinxUtils.t.sol

```diff
@@ -121,10 +121,4 @@ contract SphinxUtils_Test is Test, SphinxUtils {
 
         assertEq(uniqueValues.length, 0, "The returned array should be empty");
     }
-
-
-    function test_networkEnumSize_success() external {
-        uint256 expected = uint8(type(Network).max) + 1;
-        assertEq(numSupportedNetworks, expected);
-    }
```

Comment:
This is unrelated to the PR. I just realized we don't need to test this anymore since the networks Enum is autogenerated. I noticed and removed it b/c I also encountered an issue with this test related to compiling with `0.8.4`.